react/LocationSearch.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Input, IconSearch, IconClear } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+import styles from './LocationSearch.css'
+
+const LocationSearch: React.FC = () => {
+  const [inputValue, setInputValue] = useState<string>('')
+
+  const onClear = () => setInputValue('')
+
+  return (
+    <div className={`${styles.locationSearch} w-100`}>
+      <Input
+        label={
+          <FormattedMessage id="place-components.label.autocompleteAddress" />
+        }
+        prefix={<IconSearch color="#134CD8" />}
+        suffix={
+          inputValue.length && (
+            <span
+              role="button"
+              tabIndex={-1}
+              className="pointer"
+              onClick={onClear}
+              onKeyPress={onClear}
+            >
+              <IconClear color="#CACBCC" />
```

Comment:
How to use a tertiary color instead?

Reply:
you can use `className="c-muted-3"`

Reply:
Only works when added on the span tag, but works very well 👌